UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.API.FunctionalTests/Helpers/TestConfiguration.cs

```diff
@@ -6,8 +6,8 @@ public class TestConfiguration
     {
         protected IConfigurationRoot ConfigurationRoot;
         public FleetManagerB2BApiConfiguration fleetManagerB2BConfig = new();
-        public EssAuthorizationConfiguration EssAuthorizationConfig = new();
-        public FunctionalTestFSSApiConfiguration FssConfig = new();
+        public ESSApiConfiguration EssAuthorizationConfig = new();
+        public FSSApiConfiguration FssConfig = new();
```

Comment:
This need to be corrected wherever it referenced

```suggestion
        public ESSApiConfiguration EssApiConfiguration = new();
        public FSSApiConfiguration FssApiConfiguration = new();
```

Reply:
Updated.